frontend/test/metabase/scenarios/organization/timelines-question.cy.spec.js

```diff
@@ -257,6 +257,90 @@ describe("scenarios > organization > timelines > question", () => {
       cy.findByText("Visualization").should("be.visible");
       cy.findByLabelText("star icon").should("be.visible");
     });
+
+    it("should toggle individual event visibility", () => {
+      cy.createTimelineWithEvents({
+        timeline: { name: "Releases" },
+        events: [
+          { name: "RC1", timestamp: "2018-10-20T00:00:00Z", icon: "cloud" },
+        ],
+      });
+
+      cy.createTimelineWithEvents({
+        timeline: { name: "Timeline for collection", collection_id: 1 },
+        events: [
+          { name: "TC1", timestamp: "2016-05-20T00:00:00Z", icon: "warning" },
+        ],
+      });
+
+      visitQuestion(3);
+
+      cy.findByText("Visualization").should("be.visible");
+      cy.findByLabelText("cloud icon").should("be.visible");
```

Comment:
@nemanjaglumac,

I've checked and we do need to use `findByLabelText` to select these icons in the chart.

These event icons are rendered as svg paths, and the class names we use to select with `cy.icon` are not added to them.